test_optu.py

```diff
@@ -0,0 +1,10 @@
+import optunahub
+
+
+module = optunahub.load_module(
+    package="optuna_enhanced_visualization",
+    repo_owner="nourbc211",
+    ref="main",
+)
```

Comment:
Please do not load the package from your fork of optunahub-registry.
I think this test file can be removed since it simply loads the module.

```suggestion
module = optunahub.load_module(
    package="visualization/optuna_enhanced_visualization",
)
```